Exercise/Exercise_3/sign_in/src/Components/SignUpComponent/SignUpPassword/SignUpPassword.jsx

```diff
@@ -0,0 +1,6 @@
+const SignUpPassword = (props) => {
+    return (
+        <input className ="input" placeholder="Password" name="password" onChange={(event) => props.getPassword(event)} > </input>
```

Comment:
Why have space before </input>. This have led to error
<img width="891" alt="image" src="https://user-images.githubusercontent.com/79957186/201536260-0658fba0-dd34-4e5e-89bb-65e149e07651.png">
Please remove space